### DISCUSSION TITLE
How can i change background color property of Mantine ?

### DISCUSSION THREAD

**Discussion by dukesx** [mantinedev/mantine#197] [Q&A] [answered]
I am trying to figure out, on how I can tell Mantine to not use 7th color in dark palette but use 8th or 9th (in colors array) for background color of all components.  there is no property in Mantine provider for it. 

The issue is, there seems to be no tutorial or sample or any code that can help in this matter. What am i missing ? The docs just say that you can configure **Global Styles** or make your own,  yet there is no sample code on how to make your own or how to configure it.

**Comment by rtivital** [comment, marked as answer]
There is no way to change the position of of the color, 7th shade will always be considered as body background color in dark mode. If you do not like default Mantine dark color scheme, you can provide your own dark colors – https://mantine.dev/theming/extend-theme/

**Reply by dukesx** [reply]
Can't it be exposed somehow ? just like primary color is open to customize, i think background property and text color property should also be open to customization.

**Reply by rtivital** [reply]
You have full control over color scheme. You can customize it by changing an array of `theme.colors.dark` colors – https://mantine.dev/theming/dark-theme/#colors
